### PR TITLE
Feature/choose color & Navigation

### PR DESCRIPTION
**Pull request by Katemuya**

### CODE REVIEW

File: components/Calendar/index.js [hunk 1]
```diff
@@ -1,29 +1,34 @@
 import { Calendar as MyCalendar, momentLocalizer } from "react-big-calendar";
 import "react-big-calendar/lib/css/react-big-calendar.css";
 import styled from "styled-components";
-import Link from "next/link";
 import { useCallback, useMemo, useState } from "react";
 import { Views } from "react-big-calendar";
 import moment from "moment";
```
Comment (actyvystom) [inline components/Calendar/index.js:6]: What are you using `moment` for? This is a very outdated library, they state themselves on their website that there are better alternatives nowadays, see https://momentjs.com/docs/#/-project-status/recommendations/
As you are using already other date libraries (and `moment` does not even show up as dependency in your `package.json` 🤔), I suggest to get remove it and use the date libraries you already rely on in your dependencies (`date-fns` etc.)

**Review by actyvystom — CHANGES_REQUESTED** [review]
Well done! 👍🏻 
See a comment on `moment` and other date libraries - 
and your tests fail. Either remove all tests (the one failing came with the capstone template and is outdated due to a changed Heading component) or adjust them to run correctly. 
Cheers, Thomas

**Comment by Katemuya** [comment]
This is a bit chaotic because of many last minute and crucial changes so please bare with me. The tests fail because I removed the existing ones and I am yet to write my own.

**Comment by ArnelaKapllani** [comment]
Hi @Katemuya, you have done a really good job till now.
According to the last commits, I can't find anything to comment on so you have a green hook from me 😊

**Comment by Katemuya** [comment]
Thanks alot @ArnelaKapllani . I wish you all the best ahead 🤗

**Pull request merged**